derive_builder/tests/compile-fail/builder_setter_attrs.rs

```diff
@@ -0,0 +1,19 @@
+#![deny(unused_must_use)]
+
+#[macro_use]
+extern crate derive_builder;
+
+#[derive(Debug, PartialEq, Default, Builder, Clone)]
+pub struct Lorem {
+    ok: String,
+
+    // This foolish repr() attribute generates an unused attribute warning
+    #[builder_setter_attrs(
+        #[must_use]
```

Comment:
How does rustfmt feel about these?

Reply:
Well, it doesn't *mind* them, in the sense that it is happy to process and reformat other things in the file, including the decorated field name and type.  But rustfmt doesn't seem to want to modify the nested attribute itself.  I tried misindenting the `#[must_use]` in the test case, and rustfmt didn't fix it.  It *does* still fix the indentation of the actual field name and type.

There would be no real difficulty with accepting this syntax instead:
```
    #[builder_setter_attrs(must_use)]
```
I experimented with this and then rustfmt *did* see it and re-indent it.

I chose the syntax you see not because it was easier to implement (although it was, a bit) but because I liked the way that it makes the syntax and semantics of the contained attributes very plain.  The reader doesn't have to ask themselves questions about `#[` etc.  (That each use of `builder_setter_attrs` might contain zero or more attributes follows from the syntax; it seemed pernickety to insist on precisely one, or to insist on `builder_setter_attrs` not being repeated.)

OTOH [`cfg_attr`](https://doc.rust-lang.org/reference/conditional-compilation.html#the-cfg_attr-attribute) provides precdent for doing it the other way.  Personally I find the syntax of `cfg_attr` unnecessarily confusing.

Up to you :-).  (I think you will agree with me that we should pick *one* of these syntaxes, rather than accepting either, even though the latter might be possible in theory.)

Reply:
I certainly agree with only accepting one syntax, and I am warming to the approach you've taken. I don't find `cfg_attr` that confusing, but I could see having one `#[builder_field_attrs` being nicer than multiple for `#[builder_field_attr(...)]` declarations.

I'm going to think about this during the day today.

Reply:
Cool, thanks.  I had another thought, which is that `#[builder_field_attrs( #[serde(with...)] )]` has the advantage that it is more likely to show up in a narrowly-targeted grep.

Also, please let me know whether you'd prefer a merge, or a rebase, for (i) if you want me to change the syntax, and (ii) to deal with the textual merge conflict with #238.